### PR TITLE
Add rule forbidding more than one constant declared per statement

### PR DESCRIPTION
**Pull request by carusogabriel**
Although PSR only have this rule suggested to class properties, we
enforce it for constants as well.

### CODE REVIEW

File: tests/fixed/constants-var.php [hunk 1]
```diff
@@ -6,7 +6,13 @@
...
 
+const BAR_1 = 1;
+const BAR_1 = 2;
+
 class Bar
 {
     public const BAZ = 456;
+
+    protected const PROPERTY_1 = '1';
+    protected const PROPERTY_2 = '2';
```
Comment (carusogabriel) [inline tests/fixed/constants-var.php:17]: Spacing between constants will be proposed in a separate PR.

Spoiler: [SlevomatCodingStandard.Classes.ConstantSpacing](https://github.com/slevomat/coding-standard#slevomatcodingstandardclassesconstantspacing-).

**Comment by greg0ire** [comment]
Thanks @carusogabriel !
